pyramid_zipkin/logging_helper.py

```diff
@@ -108,7 +121,8 @@ def log_spans(self):
                 )
 
             # Collect extra annotations for server span, then log it.
-            extra_annotations = annotations_by_span_id[self.zipkin_attrs.span_id]
+            extra_annotations = annotations_by_span_id[
```

Comment:
Is there a way we can make this pre-commit hook follow the style of the directly following line? Closing square bracket on its own line. Or we can just adopt this pattern everywhere. Mixing indent styles--